ui/analyse/css/_action-menu.scss

```diff
@@ -110,6 +110,10 @@
       width: 100%;
     }
 
+    input[id='analyse-multipv'] {
+      padding: 0 0 0 0em;
+    }
```

Comment:
what is this about?

Reply:
That change allows the Multiple lines slider to visually represent zero. 

Without this particular change, the slider would be partially filled even when the multiple lines setting is set to zero. For example:
![image](https://user-images.githubusercontent.com/29963548/117000839-a7d5e800-acaf-11eb-888a-b8f1eed3e224.png)

Reply:
Probably cleaner to use `[min='0']` as the selector and add it inside the other `[type='range']` stuff. Also, you can just do `padding: 0`.